.github/workflows/dependency-audit.yml

```
@@ -0,0 +1,91 @@
+name: dependency-audit.yml
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)